### ISSUE TITLE
How can I train the model for road segmentation?

### ISSUE THREAD

**Issue by zhangchbin** [nv-tlabs/lift-splat-shoot#8]
Hi, @jonahthelion and @Kevinpgalligan,
how can I train the model for road segmentation?

**Comment by zhangchbin** [comment]
Thanks for your help.
As described in the paper
```
We deﬁne two object-based segmentation tasks and two map-based tasks. For the object segmentation tasks, we obtain ground truth bird’s-eye-view targets by projecting 3D bounding boxes into the bird’s-eye-view plane. Car segmentation on nuScenes refers to all bounding boxes of class vehicle.car and vehicle segmentation on nuScenes refers to all bounding boxes of meta-category vehicle. Car segmentation on Lyft refers to all bounding boxes of class car and vehicle segmentation on nuScenes refers to all bounding boxes with class ∈ { car, truck, other_vehicle, bus, bicycle } . For mapping, we use transform map layers from the nuScenes map into the ego frame using the provided 6 DOF localization and rasterize.
```
there are two map-based tasks, road and lane segmentation, and two object segmentation tasks, car and vehicles.
But the code only contains car segmentation.

**Comment by Kevinpgalligan** [comment]
Yeah, they don't seem to have included that part. You may have to recreate it yourself.

**Comment by zhangchbin** [comment]
Thanks for your help.

**Comment by zhangchbin** [comment]
Hi, @Kevinpgalligan and @jonahthelion ,
I have created the dataset for the road segmentation and drivable area segmentation, like this:
![@@1626147105738](https://user-images.githubusercontent.com/35215543/125385984-1e891300-e3ce-11eb-876b-b2ff64e5c5e6.png)
![@@1626147118881](https://user-images.githubusercontent.com/35215543/125385987-1f21a980-e3ce-11eb-9408-567dca66555a.png)

But if I use the original training setting, the performace is very worse. Is there any suggestion? Thanks for your help.

**Comment by sunnyHelen** [comment]
@zhangchbin Hi, how did you get the dataset for the road segmentation and drivable area segmentation? I also need these. Could you please tell about it more?

**Comment by Kevinpgalligan** [comment]
What training parameters were those? How long did you train for?  And do you mean that the performance is worse than the vehicle segmentation performance?

**Comment by berkbilir** [comment]
Hi @zhangchbin, did you manage to find the cause of the problem for low performance for road segmentation?

**Comment by linlion0311** [comment]
@zhangchbin How did you get the dataset for the road segmentation and drivable area segmentation? I also want to know , thanks.
